lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -4,4 +4,77 @@
             data-user-create-subcomment="<%= canCreateSubComment %>">
         <a href="#" class="new-post-btn" role="button"><span class="icon fa fa-edit new-post-icon"></span><%= gettext("New Post") %></a>
     </div>
+
+    <div class="team-detail">
+        <h4><%- gettext("Team Details") %></h4>
+        <% if (isMember) { %>
+            <div class="team-member">
+                <span><%- gettext("You are a member of this team.") %></span>
+            </div>
+        <% } %>
+        <div class="team-members">
+            <span class="sr"><%- gettext("Team members profile") %></span>
+            <div class="members-info"></div>
+        </div>
+
+        <div class="team-capacity">
+            <span class="sr"><%- gettext("Team capacity") %></span>
+            <span><%- membershipText %></span>
+        </div>
+
+        <div class="team-country">
+            <span class="sr"><%- gettext("Team location") %></span>
```

Comment:
I'm not sure it will always be the actual location of the team. 
"The country that team members primarily identify with." might be more accurate, as this is how we describe it in the Create Team page.  @explorerleslie Thoughts?